src/screens/Lessons/screens/Lesson/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {lessonTitleText} from 'utils/text'
+import Split from 'components/Split'
+import NextSteps from './components/NextSteps'
+import Data from './components/Data'
+
+export default ({lesson}) => (
+  <Split
+    title={lessonTitleText}
+    main={
+      <NextSteps lesson={lesson} />
+    }
+    aside={
+      <Data lesson={lesson} />
+    }
+  />
+)
```

Comment:
This view:
![image](https://cloud.githubusercontent.com/assets/5497885/22714751/2245c6ec-ed4b-11e6-935e-40d97c3433d6.png)